src/components/ClosingPriceBar/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import { Balance, TokenCode } from 'types'
+
+export interface ClosingPriceBarProps {
+  sellToken: TokenCode,
+  sellTokenPrice: Balance,
+  buyToken: TokenCode,
+}
+
+const ClosingPriceBar: React.SFC<ClosingPriceBarProps> = ({ sellToken, sellTokenPrice, buyToken }) => 
+  <div className="auctionLastPrice">
+    <small>Closing price of last auction:</small>
```

Comment:
On `PANEL 3` we have this same component but with `<small>`~~Closing~~ `price of last auction:</small>`
It would be nice to vary the text based on some kind of prop

Reply:
you know what, good call I saw your implementation of the `panel variation` in your Auction `story` and I'm gonna go with that